cpp/src/io/parquet/writer_impl.cu

```diff
@@ -1706,14 +1706,12 @@ auto convert_table_to_parquet_data(table_input_metadata& table_meta,
   // fragments with a (potentially) varying number of fragments per column.
 
   // first figure out the total number of fragments and calculate the start offset for each column
-  std::vector<size_type> frag_offsets;
+  std::vector<size_type> frag_offsets(num_columns, 0);
```

Comment:
OMG, can't suggest on the whole lambda :(
This lambda is doing two things, and it does not really need to
```
      std::exclusive_scan(
        frags_per_column.cbegin(), frags_per_column.cend(), frag_offsets.begin(), 0);
      auto const total_frags = (not frags_per_column.size().empty()) ? frag_offsets.back() + frags_per_column.back() : 0;
```

Reply:
true, I'll clean it up.